ra/ra_test.go

```diff
@@ -1135,6 +1136,24 @@ func TestAuthzRateLimiting(t *testing.T) {
 	test.AssertNotError(t, err, "NewAuthorization failed")
 }
 
+func TestAuthzFailedRateLimiting(t *testing.T) {
+	_, sa, ra, _, cleanUp := initAuthorities(t)
```

Comment:
You'll have to change this from `_, sa, ra, _, cleanup` to `_, _, ra, _, cleanup` since you don't use `sa` anywhere in this function. It looks like this is the reason why [CI is failing this PR](https://travis-ci.org/letsencrypt/boulder/jobs/285664978):
> ra/ra_test.go:1140:43: sa declared and not used